client/components/formatted-header/index.jsx

```diff
@@ -17,6 +17,7 @@ function FormattedHeader( {
 	subHeaderAlign,
 	isSecondary,
 	hasScreenOptions,
+	subHeaderAs: SubHeaderAs = 'p',
```

Comment:
Adding the prop so that we can also have the sub header as `div`. Otherwise, it is a violation to print `div` within `p`.